lib/thrift/generator/client.ex

```diff
@@ -0,0 +1,270 @@
+defmodule Thrift.Generator.ClientGenerator do
+  defmodule Framed do
+    alias Thrift.Generator.Service
+    alias Thrift.Parser.Models.Function
+
+    def generate(service_module, service) do
+      functions = service.functions
+      |> Map.values
+      |> Enum.map(&generate_function(service_module, &1))
+
+      quote do
+        defmodule Client.Framed do
```

Comment:
There's a buffered client, but it's not as simple as removing `packet: 4`. You have to do the buffering yourself.